integration/kubernetes/run_kubernetes_tests.sh

```diff
@@ -30,33 +32,45 @@ fi
 # using cri-o as the runtime.
 systemctl is-active --quiet docker || sudo systemctl start docker
 
+K8S_TEST_UNION=("k8s-replication.bats" \
+	"nginx.bats" \
+	"k8s-uts+ipc-ns.bats" \
```

Comment:
Nit: Would be good to sort this list alphabetically.

Reply:
okay~~~